<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>991</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ejercicio 1</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>831</width>
        <height>491</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>521</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre-Edad-TipoDeclase-Materia-Calificacion</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ingrese en este formato</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="te_estudiantes">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>471</width>
          <height>251</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Bton_Agregarestudiante">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>400</y>
          <width>151</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Agregar Estudiante</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Modificar</string>
       </attribute>
       <widget class="QListWidget" name="lw_listadoestudiante">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>241</width>
          <height>161</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="te_ModificarEstudiante">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>60</y>
          <width>401</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>380</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Modificar</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>20</y>
          <width>391</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre-Edad-TipoDeclase-Materia:Calificacion</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Eliminar</string>
       </attribute>
       <widget class="QLineEdit" name="le_EliminarEstudiante">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>100</y>
          <width>161</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>100</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre del Estudiante</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bton_eliminarEstudiante">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>200</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Sistema de Gestión de Estudiantes </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ejercicio 2</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Sistema de Gestión de Vehículos en una Flota </string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>771</width>
        <height>461</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Agregar Auto</string>
       </attribute>
       <widget class="QTextEdit" name="te_AgregarCarro">
        <property name="geometry">
         <rect>
          <x>123</x>
          <y>76</y>
          <width>501</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>50</y>
          <width>391</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ingrese Formato  Auto-Año-Kilometraje-mantenimiento</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bton_AgregarAuto">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>380</y>
          <width>111</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Agregar Auto</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Agregat Camion</string>
       </attribute>
       <widget class="QTextEdit" name="te_AgregarCamion">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>60</y>
          <width>431</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="bton_Agregarcamion">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>350</y>
          <width>121</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Agregar Camion</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>30</y>
          <width>331</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Camion-año-Kilometraje-Capacidad-Viajeslargos</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Ejercicio 3</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Sistema de Gestión de Empleados </string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>851</width>
        <height>441</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Agregar Empleado</string>
       </attribute>
       <widget class="QPushButton" name="bton_seleccionarEmpleado">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>130</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Seleccionar</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>141</width>
          <height>26</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>EmpleadoFijo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EmpleadoContratado</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Tipo Empleado</string>
        </property>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>40</y>
          <width>341</width>
          <height>251</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>70</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Nombre</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_NombreEmpleado">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>70</y>
            <width>113</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>130</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Edad</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_Edad">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>130</y>
            <width>113</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>180</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Salario</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="Salario">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>180</y>
            <width>113</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>20</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Fijo</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QLineEdit" name="le_Edad_2">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>120</y>
            <width>113</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>60</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Nombre</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_NombreEmpleado_2">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>60</y>
            <width>113</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>120</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Edad</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>190</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Salario</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="Salario_2">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>190</y>
            <width>113</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_18">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>20</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Contradado</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="Bton_Agregar">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>320</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Calcular Salario</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Bton_Salir">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
